### DISCUSSION TITLE
How to debug nexrender on windows?

### DISCUSSION THREAD

**Discussion by solanki21** [inlife/nexrender#949] [General]
Is there way to debug nuexrender on windows? I want to step through the code to see where the issue is. Currently I am just using win-64 binary which is not very helpful. Unfortunately I am also not getting any logs so debugging seems to be the only option to see what is going on.

**Comment by inlife** [comment]
Hey,
The easiest option for such hands-on debugging would be installing nexrender from npm, and running it as a node package. There you can do modifications to node_modules/@nexrender... packages and see the results immediately.

**Reply by solanki21** [reply]
Thank you!
